### ISSUE TITLE
Add Zenodo integration to repository

### ISSUE THREAD

**Issue by rmarkello** [rmarkello/abagen#104]
## The issue

Currently in the "Acknowledgments" section of the README we provide some guidance on citing ``abagen``:

```
...if you use this code it would be good to...provide a link back to the ``abagen`` 
repository with the version of the code used...
```

This is *fine*, but it would be great to provide some other means by which to cite ``abagen``! 

## Proposed solution

[Zenodo + GitHub](https://guides.github.com/activities/citable-code/) have some really amazing features that allow syncing a GitHub repository such that every time a new release is made it triggers [Zenodo](https://zenodo.org/) to issue a new (citeable!) [DOI](https://en.wikipedia.org/wiki/Digital_object_identifier). Setting this up and then adding a bit to the README describing how to cite the Zenodo release of ``abagen`` would be awesome!

**Comment by rmarkello** [comment]
Since Zenodo only triggers when a release is made and we want to include Zenodo in our next "official" release, I think we'll have to make an alpha or pre-release to trigger Zenodo to mint a DOI for `abagen`. Once that's minted we can add some integration of the DOI into the README and whatnot and then make an "official" release.

**Comment by rmarkello** [comment]
Once #108 is merged the following steps need to be taken:

1. Tag the latest commit with `git tag -a 0.0.2-rc.1 -m ''` to specify that it is the first release candidate for version 0.0.2 of `abagen`
2. Push the tag to GitHub with `git push --follow-tags`
3. Cut a new release for version 0.0.2-rc.1 on GitHub. Make sure to check the "This is a pre-release" box!
4. Make sure that the Zenodo integration works correctly
5. Open a new PR that:
    - Adds the Zenodo "concept" DOI badge to the README
    - Adds the `citing.rst` file to the Table of Contents in `index.rst` so it is discoverable in the documentation
    - Modifies the `citing.rst` to include the Zenodo IDs required for auto-population the `abagen` Zenodo reference in the documentation
    - Updates the "Acknowledgments" section of the README to point to the `citing.rst` page in the documentation
6. Once that PR has been merged, update the CHANGELOG by running `tools/update_changes.sh` from the root of the `abagen` repository and commit the changes (skip ci!). Tag the commit with `git tag -a 0.0.2 -m ''` and push to master.
7. Cut a new release for version 0.0.2 on GitHub (and upload it to PyPi!).
8. Profit!! Zenodo integration should be fully built in to the `abagen` documentation.

Releases in the future should be easier since all the Zenodo stuff will be all figured out :crossed_fingers: Coming up with a release checklist might be nice...
